### DISCUSSION TITLE
lists of working services

### DISCUSSION THREAD

**Discussion by Sill1000** [HomeSpan/HomeSpan#534] [Q&A]
Hi,

I tried some services form this list.

Some give some errors like Doorbell  I got the message "not managed" , the icon is the home. 
Is it disable by HK or not supported by HS ?

I also. tried the slat. Not on the list but into the HAP specification..

is there a list of services that are working ?

Thanks

**Comment by HomeSpan** [comment]
Apple continuously updates the Home App - some services seem to have been deprecated, others have been changed to function differently.  Since Apple is no longer supporting non-commercial use of Home Kit, they are not updating any of their documentation, leaving it up to us to experiment and figure out what changes they make from one iOS release to the next.

The Doorbell service is one that has been changed.  I had noticed a while back that the Home App appears to fail with a "Not Supported" message when you pair a device with the Doorbell service, but more recently users have reported that regardless of the message, the Doorbell service still seems to work.  Also, users have reported that if you add a Lock Mechanism Service, the Home App works as expected.  I have not tried this myself.
